<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTDecorationRobotToolClass</class>
 <widget class="QMainWindow" name="GTDecorationRobotToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GTDecorationRobotTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/GTDecorationRobotTool/Resources/BackGround.png);</string>
    </property>
    <widget class="QGTToolButton" name="toolButton_Login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;
border-radius:16px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
     </property>
     <property name="text">
      <string>装修机器人</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QGTToolButton" name="toolButton_MiniSize">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QGTToolButton" name="toolButton_Exit">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>127</y>
       <width>1024</width>
       <height>640</height>
      </rect>
     </property>
     <widget class="QGTFunctionalDemoPage" name="page_FunctionalDemo">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/PaintRobotTool/Resources/ui_BaseSet.png);</string>
      </property>
     </widget>
     <widget class="QGTModularControlPage" name="page_ModularControl">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/PaintRobotTool/Resources/ui_BaseSet.png);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/PaintRobotTool/Resources/ui_BaseSet.png);</string>
     </property>
     <widget class="QGTToolButton" name="toolButton_Select_FunctionalDemo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:30px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(135, 206, 235);</string>
      </property>
      <property name="text">
       <string>演示</string>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
     <widget class="QGTToolButton" name="toolButton_Select_ModularControl">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:30px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(135, 206, 235);
</string>
      </property>
      <property name="text">
       <string>控制</string>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGTToolButton" name="toolButton_BaseSet">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QGTToolButton" name="toolButton_Auto_Run">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGTToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">Commen/qgttoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QGTFunctionalDemoPage</class>
   <extends>QWidget</extends>
   <header location="global">qgtfunctionaldemopage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QGTModularControlPage</class>
   <extends>QWidget</extends>
   <header location="global">qgtmodularcontrolpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gtdecorationrobottool.qrc"/>
 </resources>
 <connections/>
</ui>
